Domain Layer/Product.java

```diff
@@ -0,0 +1,31 @@
+// Product class representing individual products
```

Comment:
the package name should be in all lower case, can contain digits but must not start with a digit, and "." should be used as a separator, not spaces.  Please change the following package names DOMAIN LAYER, EXCEPTION , MAIN LAYER , REPOSITORY LAYER, SERVICELAYER